pkg/prowgen/prowgen.go

```diff
@@ -236,6 +236,24 @@ func generatePostsubmitsForPromotion(jobBaseBuilderFactory func() *prowJobBaseBu
 	for _, opt := range options {
 		opt(opts)
 	}
+
+	if info.Config.MultiArch {
+		jobBaseBuilder := jobBaseBuilderFactory()
```

Comment:
What's the issue with the current implementation? I don't understand why we need to re-implement everything just to include a cluster.

Reply:
@danilo-gemoli Check https://github.com/openshift/ci-tools/blob/master/pkg/prowgen/jobbase.go#L53-L100